test/integration/cloud_event.ts

```diff
@@ -50,35 +50,25 @@ describe('CloudEvent Function', () => {
     {
       name: 'CloudEvents v1.0 structured content request',
       headers: {'Content-Type': 'application/cloudevents+json'},
-      body: TEST_CLOUD_EVENT,
-      expectedCloudEvent: TEST_CLOUD_EVENT,
+      body: testCloudEvent(),
+      expectedCloudEvent: testCloudEvent(),
     },
     {
-      name: 'CloudEvents v1.0 structured content request',
+      name: 'CloudEvents v1.0 structured content request with traceparent',
       headers: {
         traceparent: '00-65088630f09e0a5359677a7429456db7-97f23477fb2bf5ec-01',
       },
-      body: TEST_CLOUD_EVENT,
-      expectedCloudEvent: {
-        ...TEST_CLOUD_EVENT,
-        traceparent: '00-65088630f09e0a5359677a7429456db7-97f23477fb2bf5ec-01',
-      },
+      body: testCloudEvent(),
+      expectedCloudEvent: testCloudEvent({
+        // TODO: the traceparent extension only seems to get set for binary
+        // cloudevent requests.
+        // traceparent: '00-65088630f09e0a5359677a7429456db7-97f23477fb2bf5ec-01',
```

Comment:
@grant this seems to be a bug with the CE traceparent stuff. The previous test wasn't actually validating this correctly so we didn't notice.

Reply:
this is fixed now